doc/programming/parsing.rst

```diff
@@ -861,6 +861,16 @@ properties:
     the module level; use ``Null`` to reset the property, i.e., not skip
     anything.
 
+``%synchronize-at = EXPR;``
+    Specifies a literal to synchronize on if the unit is used as a
+    synchronization point during :ref:`error recovery <error_recovery>`.
+    The literal is left in the input stream.
+``%synchronize-after = EXPR;``
```

Comment:
Formatting nit: let's add empty lines between the properties (between the `%skip` props above too).